new-site/src/docs/about/contributing/before-contributing.mdx

```diff
@@ -0,0 +1,51 @@
+---
+slug: '/about/contributing/before-contributing'
+title: 'Before contributing'
+---
+
+import { Link, Notification } from 'hds-react';
+import LeadParagraph from '../../../components/LeadParagraph';
+import Image from '../../../components/Image';
+
+# Before contributing
+
+<LeadParagraph>
+    You can contribute content to the HDS in different ways. Before you start contributing to HDS, please refer to following guidelines.
+</LeadParagraph>
+
+Every component in HDS includes three parts:
+1. Design 
+2. Implementation 
+3. Documentation 
+
+You may contribute in any part of these but all three parts have to be implemented and validated by HDS team before a feature can be released.
+
+<Notification size="small" label="HDS team response time" type="alert" className="siteNotification" style={{marginBottom: 'var(--spacing-m)'}}>
+The process of adding component to HDS can take a while. New feature request and proposed designs are first evaluated and prioritised on HDS roadmap. If the need for a feature is urgent, discuss this with HDS team.
+</Notification>
+
+The easiest way to start contributing to HDS is joining the City of Helsinki Slack channel (<Link size="M" href="https://helsinkicity.slack.com/archives/CHCV3KTHA" external>#designsystem</Link>) and send us a message.
+
+## Tools needed
+Depending on the feature you want to be working on, you will need access to at least some of these tools:
+- <Link size="M" href="https://www.sketch.com/" external>Sketch</Link>
+- City of Helsinki organisation in <Link size="M" href="https://www.abstract.com/" external>Abstract</Link>
+- City of Helsinki Slack (channel: <Link size="M" href="https://helsinkicity.slack.com/archives/CHCV3KTHA" external>#designsystem</Link>)
+- <Link size="M" href="https://github.com/City-of-Helsinki/helsinki-design-system" external>HDS GitHub repository</Link>
+
+## Contribution process
+When faced with a need for a component or feature, you can ask yourself a few simple questions to decide whether you should propose a new component or feature to be added in HDS, or design a custom implementation:
+
+<Image src="../../../hds-contribution.png" alt="How to contribute to Helsinki Design System" style="max-width:100%" viewable />
```

Comment:
The image urls do not work. I tried to solve this but maybe they have to be fixed in another PR

Reply:
Images are not passed through to the `/public` folder. That's seems to be the reason.